packages/clay-color-picker/README.md

```diff
@@ -0,0 +1 @@
+# clay-color-picker
```

Comment:
We can fill in `README.md` with the same information we have in the other components. See [ClayDatePicker](https://github.com/liferay/clay/blob/2e847b45db7e308799ed682029a995c1ab16f81f/packages/clay-date-picker/README.md)